### ISSUE TITLE
Ai distracted by unfinished turrets

### ISSUE THREAD

**Issue by Silvergnom** [jsettlers/settlers-remake#631]
Hello fellas,

The Ai wants to claim unfinished towers and castles.
So when you put a tower somewhere and you stop the building process, the Ai army will still try to capture this one and is distraced forever.
Easy way to sneake around his army and have your base safe forever.
Is there a method like .isFinished? Should be easy then to get rid of this weakness.

Kind regards,

Silvergnom

**Comment by PaweX** [comment]
Yes, still not working in the version: 0af21e3b and can break the entire game:
![6-AI-attacks_unfinished_tower](https://user-images.githubusercontent.com/3579486/54564856-263a7780-49cd-11e9-8c73-658684eb370a.jpg)
